packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -108,7 +108,7 @@ def __init__(self, **kwargs):
             and self.crypto_store.crypto_objects.get("fetchai", None) is not None
         ):  # pragma: no cover
             key = cast(FetchAICrypto, self.crypto_store.crypto_objects["fetchai"])
-        elif key_file is not None:
+        elif key_file is not None:  # pragma: no cover
```

Comment:
on my local machine, with file (missing from develop) `test_errors.py`, this is already covered by  `tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py::TestLibp2pClientConnectionFailureConnectionSetup::test_empty_nodes`
(https://github.com/fetchai/agents-aea/blob/feature/libp2p-improve-coverage/tests/test_packages/test_connections/test_p2p_libp2p_client/test_errors.py#L77)